db/post_migrate/20200430173433_remove_user_id_from_invites.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'migration/column_dropper'
+
+class RemoveUserIdFromInvites < ActiveRecord::Migration[5.2]
+  DROPPED_COLUMNS ||= {
+    invites: %i{
+      user_id
+      redeemed_at
+    }
+  }
+
+  def up
+    execute <<SQL
+      INSERT INTO invited_users (
+        user_id,
+        invite_id,
+        redeemed_at,
+        created_at,
+        updated_at
+      )
+      SELECT user_id, id, redeemed_at, created_at, updated_at
+      FROM invites WHERE user_id IS NOT NULL
+SQL
+
+    DROPPED_COLUMNS.each do |table, columns|
```

Comment:
ColumnDropper is only useful if you need `.mark_readonly` or you are disabling ddl transaction. In this case you are doing neither, so you can simply drop the column. also be sure to add something like this: 

https://github.com/discourse/discourse/blob/a2d939608d5b3adc7f037dc0ca38e7d1f6b895f6/app/models/topic.rb#L13-L14

Reply:
Done.